python/python-hello-world/src/app.py

```diff
@@ -1,22 +1,36 @@
+# Copyright 2020 Google LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     https://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 """
 A sample Hello World server.
 """
 import os
-from flask import Flask
+import requests
+
+from flask import Flask, render_template
 
 # pylint: disable=C0103
 app = Flask(__name__)
 
+
 @app.route('/')
 def hello():
     """Return a friendly HTTP greeting."""
-    message = "Hello World"
-    return message
+    message = "It's running!"
 
-if __name__ == '__main__':
-    server_port = os.environ.get('PORT')
-    if server_port is None:
-        print("error: PORT environment variable not set")
-        exit(1)
+    return render_template('index.html', message=message)
```

Comment:
You're passing in a message to your template, but your template doesn't seem to have any variables to interact with this message?  

Do we need the template?